cloud/datasync/package.json

```diff
@@ -35,10 +35,12 @@
     "@types/proxyquire": "^1.3.27",
     "del-cli": "^1.0.0",
     "mocha": "^3.4.2",
+    "mocha-typescript": "^1.1.5",
```

Comment:
-1 for using this, I had investigated it before. It breaks the goal that our test suites be usable from alternative, plain js implementations.

See the [js template](https://github.com/feedhenry-raincatcher/raincatcher-core/tree/master/templates/javascript-usage) for an example of JS code using the compiled TS tests.